src/crypto/crypto_keys.cc

```diff
@@ -871,7 +871,6 @@ void KeyObjectData::MemoryInfo(MemoryTracker* tracker) const {
 }
 
 std::shared_ptr<KeyObjectData> KeyObjectData::CreateSecret(ByteSource key) {
-  CHECK(key);
```

Comment:
Why is a zero length buffer key data treated as a nullptr?

Reply:
It might be because `malloc(len)` for `len = 0` is allowed to return a `nullptr`. The standard only requires the returned pointer to be valid for `len` bytes, which `nullptr` fulfills when `len = 0`.

Reply:
Do you have a suggestion that would allow us to keep this check?